app/controllers/display_task_controller.rb

```diff
@@ -1,12 +1,16 @@
 class DisplayTaskController < ApplicationController
   def index
     @all_tasks = Task.all
+    @all_people = Person.all
+          
     render :index
   end
 
   def links
     @id = params[:id]
```

Comment:
You want to be sure that you're only exposing instance variables that are required for your views. Since the `@id` variable here is only used to retrieve the task in the following line, and not used in the view, this should be a local variable rather than an instance variable.